File: backend/src/models/user.model.js
const db = require("../database/db");

const createUser = (user, callback) => {
  const sql = `
    INSERT INTO tbd_usuarios (nombre, email, password, token, telefono)
    VALUES (?, ?, ?, ?, ?)
  `;

  db.query(
    sql,
    [user.nombre, user.email, user.password, user.token, user.telefono || null],
    callback
  );
};

module.exports = { createUser };

File: API/models/user_model.py
from sqlalchemy import Column, Integer, String, Boolean, DateTime
from config.database import Base

class UserModel(Base):
    __tablename__ = "tbd_usuarios"

    id = Column(Integer, primary_key=True, index=True)
    nombre = Column(String(50))
    email = Column(String(100), unique=True, index=True)
    password = Column(String(255))
    rol = Column(String(20), default="user")
    confirmado = Column(Boolean, default=False)
    token = Column(String(255), nullable=True)
    current_session = Column(String(255), nullable=True)
